sympy/solvers/solveset.py

```diff
@@ -2411,7 +2410,63 @@ def solvify(f, symbol, domain):
 ###############################################################################
 
 
-def linear_coeffs(eq, *syms, **_kw):
+def _le2d(e, s, strict, _redo=True):
```

Comment:
This function looks quite similar to `_linear_eq_to_dict`. Is there a need to have two separate functions?